### PR TITLE
DBZ-4071 Pass recipient mail address as a param to pipeline job

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-4071

### CODE REVIEW

File: jenkins-jobs/pipelines/build-debezium-images.groovy [hunk 1]
```diff
@@ -182,6 +181,6 @@ node('Slave') {
             }
         }
     } finally {
-        mail to: ${MAIL_ID}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
+        mail to: ${MAIL_TO}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
```
Comment (jpechane) [inline jenkins-jobs/pipelines/build-debezium-images.groovy:184]: `MAIL_TO` is variable so it should be referedd as `MAIL_TO` only. This syntax is only when embedded inside string.

Comment (ani-sha) [reply jenkins-jobs/pipelines/build-debezium-images.groovy]: Thanks for the clarification. Fixed it.

**Review by gunnarmorling — APPROVED** [review]
LGTM, but will leave it to @jpechane to merge this one :) Thanks!

**Comment by jpechane** [comment]
@ani-sha Applied, thanks a lot.

**Pull request merged**